### ISSUE TITLE
[Feature Request] Statusbar button alignment

### ISSUE THREAD

**Issue by lonix1** [lbauskar/kanban#4]
The button on the statusbar is shown in an arbitrary location.

It would be best if it would be shown, by default, on the 1) extreme left or extreme right when left aligned, or 2) extreme left or extreme right when right aligned.

It would be even better if we could configure that setting manually.

There's an API for setting a statusbar item's priority. See [this extension](https://github.com/Fooxly/vscode-provision-bar/blob/6a77ebd6b20719b557aa55c688e09e04060eb056/src/main/Hub.ts#L76) for an example. It also exposes a setting so we can set the priority manually.

**Comment by lbauskar** [comment]
I agree with you that there should be an option for the button to be on the left or right. I'm still not certain if setting the priority is worth having as its own discrete option.

Regardless, I've implemented both right now on my local build. I'm gonna see how intuitive the priority setting is and how many people use it, then I'll get back to you.
